contrib/database/sql/stmt.go

```diff
@@ -25,8 +27,13 @@ type tracedStmt struct {
 // Close sends a span before closing a statement
 func (s *tracedStmt) Close() (err error) {
 	start := time.Now()
+	span := s.tryStartTrace(s.ctx, queryTypeClose, "", start, nil)
```

Comment:
Any reason to switch to `tryStartTrace` here? It looks like there's no sql commenting being done